client/state/analytics/reducer.js

```diff
@@ -3,11 +3,19 @@
  */
 import {
 	ANALYTICS_TRACKING_ON,
+	DESERIALIZE,
+	SERIALIZE,
 } from 'state/action-types';
 
 export const analyticsTracking = ( state = false, { type } ) => {
-	if ( ANALYTICS_TRACKING_ON === type ) {
-		return true;
+	switch ( type ) {
+		case ANALYTICS_TRACKING_ON:
+			return true;
+
+		// We don't want to persist across reloads
+		case SERIALIZE:
+		case DESERIALIZE:
+			return false;
 	}
```

Comment:
note that we can just opt-out of serialization instead of persisting a small value, which would definitely be preferred. I think this happens by returning `null`

cc: @gwwar